repositories/rhel-8.3.json

```diff
@@ -2,24 +2,24 @@
     "aarch64": [
         {
             "id": "baseos",
-            "baseurl": "http://download.devel.redhat.com/rhel-8/rel-eng/RHEL-8/latest-RHEL-8.3/compose/BaseOS/aarch64/os",
+            "baseurl": "https://cdn.redhat.com/content/dist/rhel8/8/aarch64/baseos/os",
```

Comment:
8.3 is not yet released. Let's keep using the internal URLs until we thought about a good solution.

Reply:
This is fixed